lib/licenses.nix

```diff
@@ -546,6 +546,18 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
     fullName = "Non-Profit Open Software License 3.0";
   };
 
+  nvidiaCuda = {
+    fullName = "CUDA Toolkit Supplement to Software License Agreement for NVIDIA Software Development Kits";
```

Comment:
Maybe we should use the same name as conda does, "CUDA Toolkit End User License Agreement (EULA)", cf https://anaconda.org/conda-forge/cudatoolkit. I think we should also include the same notice they say they got NVidia to review:

> By downloading and using the packages, you accept the terms and conditions of the CUDA EULA - https://docs.nvidia.com/cuda/eula/index.html

I'd like this to be displayed at https://search.nixos.org/packages?channel=unstable&show=cudaPackages.cudatoolkit&size=1&sort=relevance&type=packages&query=cudatoolkit